mautic/mautic
# Fix elFinder paste image from search

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
This PR fixed insert image  after search in elFinder.




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Try use search feature in elFinder and select any finded image from subdirectory
3. You should see error

![image](https://user-images.githubusercontent.com/462477/138585236-4c7486a4-16ce-4410-bba3-2af9ac6ea06b.png)
